src/fixtures/geometryBox.js

```diff
@@ -0,0 +1,17 @@
+import {geometryBoxes, nonStandardKeywords} from '../validators/isGeometryBox';
+import globals from '../util/globals';
+import boxes from './box';
+
+export default {
+    valid: [
+        ...geometryBoxes,
+        ...nonStandardKeywords,
+        ...boxes.valid,
+        ...globals,
```

Comment:
Again, the tests for the global keywords are automatically generated, this isn't required. 😄